### DISCUSSION TITLE
Duplicated Github Checks

### DISCUSSION THREAD

**Discussion by fengwei-tian** [community/community#34236] [Actions]
Hi team, 

We have a CI github action that needs to be run for both pull_request and pull_request_review.

So we set this action in the `Require status checks to pass before merging` session.

However, there will be case that the status for pull_request is not in-sync with the status for pull_reqeust_review and we only require one of them to pass for PR merge.
Could you please let us know how to bypass the check when either of them pass in the PR. Thanks!

![Screen Shot 2022-09-26 at 11 27 46 AM](https://user-images.githubusercontent.com/87677672/192353777-066936dd-094b-478f-803c-eb3f73adacf7.png)

**Comment by chenzhao0613** [comment]
Hello @fengwei-tian , did you found any workaround to this issue?
